src/Tizen.Network.Connection/Tizen.Network.Connection/ConnectionTypes.cs

```diff
@@ -464,6 +464,10 @@ public enum WiFiSecurityType
         /// EAP
         /// </summary>
         Eap = 4,
+        /// <summary>
+        /// WPA3-SAE
+        /// </summary>
+        Wpa3Sae = 5,
     }
```

Comment:
This task is for adding WPA3 related enum, so we need to add the following 2 enums.
These are the security types added in WPA3 spec.
```
typedef enum {
...
	WIFI_MANAGER_SECURITY_TYPE_SAE = 6,         /**< SAE (Since 5.5) */
	WIFI_MANAGER_SECURITY_TYPE_OWE = 7,         /**< OWE (Since 5.5) */
...
} wifi_manager_security_type_e;
```
Please match the values and enums with the CAPI.
The values that the CAPI actually returns are the above values.

Reply:
I have modified the PR as per comments.